publish.css

```diff
@@ -1,23 +1,112 @@
 /* This Theme is called "Reverie" and Obsidian Theme created by Santi Younger */
-
-Last Update:
-2021-03-11 12:06
-
 @import url('https://fonts.googleapis.com/css2?family=Modern+Antiqua&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Raleway&display=swap');
 
-:root {
-  --font-monospace: 'Source Code Pro', monospace;
-
+body {
+  --font-monospace: 'Source Code Pro';
   --wws-font-modern: "Modern Antiqua";
-
   --wws-font-raleway: "Raleway";
-}
-   
+  --wws-font-text: 'Modern Antiqua';
+  --wws-font-monospace: 'Source Code Pro', Jetbrains Mono;
+  --wws-header-font: 'Modern Antiqua';
+  --wws-header-font-caps-variant: normal;
+  --wws-font-tags: var(--wws-header-font);
+  --wws-font-callout-title: var(--wws-header-font);
+
+  --h1-font: var(--wws-header-font);
+  --h2-font: var(--wws-header-font);
+  --h3-font: var(--wws-header-font);
+  --h4-font: var(--wws-header-font);
+  --h5-font: var(--wws-header-font);
+  --h6-font: var(--wws-header-font);
+  
+  --font-text-theme: var(--wws-font-text);
+  --font-monospace-theme: var(--wws-font-monospace);
+  --font-interface-theme: var(--wws-font-text);
+  --font-mermaid: var(--wws-font-monospace);
+  }
+
 .published-container {
-     --font-text-theme: var(--wws-font-modern);
-     --page-title-variant: var(--wws-font-raleway);
-}
+  /* These are the Default Ones that Come with App.css */
+
```

Comment:
Added the rest of these, that way you can further fine tune your text

Reply:
Outstanding!